RecoLocalMuon/CSCRecHitD/python/cscRecHitD_cfi.py

```diff
@@ -4,73 +4,25 @@
 
 # parameters for CSC rechit building
 from RecoLocalMuon.CSCRecHitD.cscRecHitD_cff import *
-csc2DRecHits = cms.EDProducer("CSCRecHitDProducer",
-    #
-    #    Parameters for coordinate and uncertainty calculations
-    #    Data and MC parameters are (still) different
-    #    Needs tuning
-    #
-    cscRecHitDParameters,
-    #
-    #    Parameters for strip hits
-    #
-    CSCStripPeakThreshold = cms.double(10.0),
-    CSCStripClusterChargeCut = cms.double(25.0),
-    CSCStripxtalksOffset = cms.double(0.03),
-    #
-    #    How to find SCA peak time?
-    #                              
-    UseAverageTime = cms.bool(False),
-    UseParabolaFit = cms.bool(False),
-    UseFivePoleFit = cms.bool(True),                       
-    #
-    #    Parameters for wire hits
-    CSCWireClusterDeltaT = cms.int32(1),
-    #
+import RecoLocalMuon.CSCRecHitD.configWireTimeWindow_cfi as _mod
```

Comment:
@ptcox this PR aims at reusing the default configuration as defined in the `fillDescription` method of `CSCRecHitDProducer`, see item 6 in https://twiki.cern.ch/twiki/bin/view/CMS/OpenRecoTasks#Review_and_optimization_of_t_AN1).

In this case, by removing the un-modified parameters one also removes the comment lines inserted, and possibly an easy modification of the configuration. Therefore, I'd be tempted to ask to revert the second part of this PR and limit it to the part in which the configuration is cloned from the "default" one, and parameters are set modifying the original ones, i.e. what coded in the [first commit](https://github.com/cms-sw/cmssw/pull/33354/commits/35cc9d02c3d606fd5465a7a66db56df7d54d6795) of this PR

Before asking @jeongeun to revert the second commit: do you have any suggestion or comment, Tim? Would you stick on the "thinner" configuration resulting from the current version of this PR, instead?